src/main/groovy/org/arquillian/spacelift/gradle/android/AndroidSdkInstallation.groovy

```diff
@@ -251,6 +257,19 @@ class AndroidSdkInstallation extends BaseContainerizableObject<AndroidSdkInstall
         return new File((File) project['spacelift']['installationsDir'], "${getProduct()}/${getVersion()}/${getFileName()}")
     }
 
+    private Map<String, String> getAndroidEnvironmentProperties() {
+        Map<String, String> envProperties = new HashMap<String, String>();
+        
+        File androidHome = getHome()
+        
+        envProperties.put("ANDROID_HOME", androidHome.absolutePath)
+        envProperties.put("ANDROID_SDK_HOME", androidHome.parentFile.absolutePath)
```

Comment:
well ... by default it puts .android to your $HOME

when you have ANDROID_HOME in spacelift/workspace/android-sdk-home, where would you like to see .android then? I think it this particular scenario, in spacelift/workspace/.android.

it is true that it is accidentally your workspace but it could be whatever, depends on where is getHome().